application-github-statistics-ui/src/main/resources/GitHubStats/CommittersMacro.xml

```diff
@@ -20,75 +20,228 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
 -->
 
-<xwikidoc>
+<xwikidoc version="1.3" reference="GitHubStats.CommittersMacro" locale="">
```

Comment:
in my past commits, I've removed all the changes done by the export because this extension has to work with XWiki 6.4. I don't know version 1.3 of the XAR format is supported in XWiki 6.4 so I took the safe root and discarded the export changes brought by recent versions of XWiki...